documentation/breaking-changes/breaking-changes-tool-help.md

```diff
@@ -0,0 +1,1392 @@
+
+```powershell
+Remove-SomeObject -WhatIf
+Remove-SomeObject -Confirm
+```
+
+### Remediation
+
+_Make sure the cmdlet '`<cmdlet>`' implements SupportsShouldProcess._
+
+## 1040 - Removed SupportsPaging
+
+### Description
+
+_The cmdlet '`<cmdlet>`' no longer implements SupportsPaging._
+
+When a cmdlet that previously implemented `SupportsPaging` no longer does, that is a breaking change. Users will no longer be able to use the `First`, `Skip`, and `IncludeTotalCount` parameters with the cmdlet.
+
+### Remediation
+
+_Make sure the cmdlet '`<cmdlet>`' implements SupportsPaging._
+
+## 1050 - Removed Parameter Set
+
+### Description
+
+_The parameter set '`<parameterSet>`' for cmdlet '`<cmdlet>`' has been removed._
+
+When a parameter set for a cmdlet has been removed, that is a breaking change. Existing scripts that use the removed parameter set will no longer work.
+
+### Remediation
+
+_Add parameter set '`<parameterSet>`' back to cmdlet '`<cmdlet>`'._
+
+## 1060 - Changed Default Parameter Set
+
+### Description
+
+_The parameter set '`<parameterSet>`' for cmdlet '`<cmdlet>`' is no longer the default parameter set._
+
+When the default parameter set is changed, that is a breaking change. Existing scripts that rely on the default parameter set will no longer function the same since they will now be using a different parameter set.
+
+For example, if we had a cmdlet `Add-SomeObject` whose default parameter set was `Foo`
+
+```cs
+[Cmdlet(VerbsCommon.Add, "SomeObject", DefaultParameterSetName = "Foo")]
+public class AddSomeObject : Cmdlet
+{
+    [Parameter(
+        ParameterSetName = "Foo",
+        Mandatory = false)]
+    public string Foo { get; set; }
+
+    [Parameter(
+        ParameterSetName = "Bar",
+        Mandatory = false)]
+    public string Bar { get; set; }
+
+    public override void ExecuteCmdlet()
+    {
+        ...
+    }
+}
+```
+
+but the default parameter set was changed to `Bar`
+
+```cs
+[Cmdlet(VerbsCommon.Add, "SomeObject", DefaultParameterSetName = "Bar")]
+```
+
+this will cause a change in functionality of the cmdlet and a call to `Add-SomeObject` with no parameters will not work as expected.
+
+Also, if the default parameter set is removed entirely from this cmdlet, this will cause an exception to be thrown whenever `Add-SomeObject` is called with no parameters since PowerShell will not be able to determine which parameter to use.
+
+### Remediation
+
+_Change the default parameter for cmdlet '`<cmdlet>`' back to '`<parameterSet>`'._
+
+## 1070 - Changed Output Element Type
+
+### Description
+
+_The element type for the output has been changed from `'<oldElementType>'` to `'<newElementType>'`._
+
+When the type of the output is an array, and the element type of that array has been changed, that is a breaking change. 
```

Comment:
Probably want to mention that types need to have the same proeprties, not necessarily the same names